components/Home/Home.jsx

```diff
@@ -173,15 +65,25 @@ export default () => {
       <Image
         source={require('../../assets/logo.png')}
         style={{
-          width: 215, height: 215, position: 'absolute', right: 10, top: 20,
+          width: 215,
+          height: 215,
+          position: 'absolute',
+          right: 10,
+          top: 20,
         }}
       />
-      <View style={{ marginBottom: '10%', marginTop: '5%' }}>
-        <WelcomeTitle>Hello</WelcomeTitle>
-        <WelcomeTitle numberOfLines={1}>
-          {welcomeName}
-          !
-        </WelcomeTitle>
+      <View style={{ marginBottom: '10%', marginTop: '5%', marginLeft: '5%' }}>
```

Comment:
I think we made the standard to style the components at the top and not inline, not 100% sure if required though